src/mlpack/core/tree/rectangle_tree/no_auxiliary_information.hpp

```diff
@@ -99,6 +99,22 @@ class NoAuxiliaryInformation
   }
 
   /**
+   * The R++ tree requires to split the maximum bounding rectangle of a node
+   * that is being split. This method is intended for that.
```

Comment:
It might be useful to add that this method is only necessary for an `AuxiliaryInformationType` that is being used in conjunction with `RPlusTreeSplit`.